site/content/docs/5.1/about/brand.md

```diff
@@ -41,83 +41,39 @@ Boosted uses —and recommends to use— **a single SVG file for both logos**, s
-      </figure>
-    </div>
-  </div>
-  <div class="col">
-    <div class="ratio ratio-16x9">
-      <figure class="figure d-flex border border-1">
-        <div class="figure-img m-auto d-inline-flex align-items-baseline">
-          <img src="/docs/{{< param docs_version >}}/assets/brand/orange-logo.svg" width="50" height="50" alt="" aria-hidden="true" loading="lazy">
-          <span class="h5 ms-2 mb-0 align-self-end">
-            <span class="visually-hidden">Orange </span>
-            Business<br>Services
-          </span>
-        </div>
-        <figcaption class="figure-caption fw-bold text-body position-absolute">OBS logo left-aligned with black descriptor</figcaption>
-      </figure>
-    </div>
-  </div>
-  <div class="col">
-    <div class="ratio ratio-16x9">
-      <figure class="figure d-flex bg-dark">
-        <div class="figure-img m-auto d-inline-flex align-items-baseline">
-          <span class="h5 me-2 mb-0 align-self-end text-end">
-            <span class="visually-hidden">Orange </span>
-            Business<br>Services
-          </span>
-          <img src="/docs/{{< param docs_version >}}/assets/brand/orange-logo.svg" width="50" height="50" alt="" aria-hidden="true" loading="lazy">
-        </div>
-        <figcaption class="figure-caption fw-bold text-body position-absolute">OBS logo left-aligned with white descriptor</figcaption>
-      </figure>
-    </div>
-  </div>
+Orange Business Services has its own logo that contains the Orange logo and the "Business Services" text. This set is a logo by itself, must be considered as a single SVG file and must not be built manually by assembling an image and some texts.
+
+<div class="row row-cols-md-2 mt-4 mb-5">
   <div class="col">
-    <div class="ratio ratio-16x9">
-      <figure class="figure d-flex border border-1">
-        <div class="figure-img m-auto d-inline-flex align-items-baseline">
-          <span class="h5 me-2 mb-0 align-self-end text-end">
-            <span class="visually-hidden">Orange </span>
-            Business<br>Services
-          </span>
-          <img src="/docs/{{< param docs_version >}}/assets/brand/orange-logo.svg" width="50" height="50" alt="" aria-hidden="true" loading="lazy">
-        </div>
-        <figcaption class="figure-caption fw-bold text-body position-absolute">OBS logo left-aligned with black descriptor</figcaption>
-      </figure>
+    <div class="ratio ratio-1x1">
+    <figure class="figure d-flex bg-dark">
+      <img src="/docs/{{< param docs_version >}}/assets/brand/OBS-logo.svg" class="figure-img img-fluid m-auto" width="220" height="220" role="img" alt="Orange" loading="lazy">
+      <figcaption class="figure-caption fw-bold text-body position-absolute">Orange Business Services logo</figcaption>
+    </figure>
     </div>
   </div>
 </div>
 
-### Using semantic markup
+{{< svg-docs file="img/OBS-logo.svg" >}}
+
+## Build your own logo
```

Comment:
I'd explain better this part if kept, by displaying an image with a built logo.

Reply:
Also I would have put a visual example (as there was before) even if it is a bit redundant with the OBS logo

Reply:
I'm not sure this part is even useful because you have on one hand mandatory SVG logos and on the other hand, in the global headers "Logo + title" and "Logo + title on 2 lines". Since the logo is only used in the global headers, I don't know if this part is relevant or not, in which case it could be used.